concent_api/verifier/utils.py

```diff
@@ -52,30 +52,41 @@ def prepare_storage_request_headers(file_transfer_token: message.FileTransferTok
 
 
 def store_file_from_response_in_chunks(response: requests.Response, file_path: str):
-    with open(file_path, 'x') as f:
+    with open(file_path, 'wb') as f:
         for chunk in response.iter_content():
             f.write(chunk)
 
 
-def run_blender(scene_file, output_format, script_file=''):
+def run_blender(scene_file, output_format, frame_number, script_file=''):
+    output_format = adjust_format_name(output_format)
     return subprocess.run(
         [
             "blender",
-            "-b", f"{scene_file}",
+            "-b", f"{settings.VERIFIER_STORAGE_PATH}/{scene_file}",
```

Comment:
`generate_verifier_storage_file_path` can be used here.

Reply:
done